### ISSUE TITLE
what contribute to the accuracy increase in mobilenet quantize?

### ISSUE THREAD

**Issue by dreambear1234** [Tencent/PocketFlow#145]
Uniform Quantization:
In you README, the accuracy of mobilenet-v1/v2 are all increased by 0.4,
As I know, use quantize api of tensorflow will degenerate accuracy(maybe 1 point)!
How did you achieve this? Incredible!
Is it all the contribution of distillation?

**Comment by jiaxiang-wu** [comment]
By "quantize api", which are you refering to, quantization-aware training or post-training quantization? The latter will lead to degenerated accuracy, but we are using the first one.
How to reproduce: use `UniformQuantTFLearner` with `--enbl_dst` option enabled. Distillation loss is helpful in reducing the accuracy degradation, but 8-bit uniform quantization should have very significant accuracy loss in the first place.

**Comment by dreambear1234** [comment]
@jiaxiang-wu , Thank you for your reply
'quantize api' means quantization-aware training, accuracy is found from here, 'https://github.com/tensorflow/tensorflow/tree/master/tensorflow/contrib/quantize',
In this page, accuracy is decreased by 1%.
1. Can I understand that 0.4% accuracy increase in your code all come from distillation ? (it means that 1.4(1% + 0.4%) accuracy is achieved by distillation !!)

2. By the way, in distillation training of mobilenet, who is teacher model ? (resnet101 or VGG or just mobilenet original float model). Does it differs?

Appreciate for your help!

**Comment by dreambear1234** [comment]
@jiaxiang-wu ,
1. In the situation of quantization-aware training. In that link,  learning_rate is a constant (1e-4), your learning_rate is 0.045 * 0.01 which will decrease by epochs, I don't think there are too many difference.

2. As your opinion, '8-bit uniform quantization should have very significant accuracy loss', Distillation get such a great job who reach better accuracy than float model ! It exceed the accuracy of teacher network which is amazing !

3. Another question, How many GPU is used when you quantize these model ?

Thank you for your help sincerely !

**Comment by jiaxiang-wu** [comment]
1. We have not tested the constant learning rate schedule. It is hard to tell how much accuracy difference will be caused by these two schedules.
2. Sorry, that's a typo. I want to say "8-bit uniform quantization **should not** have very significant accuracy loss". Training with distillation can often exceed the teacher network's performance. For instance, if you train a ResNet-20 model with another pre-trained ResNet-20 model as the teacher, even though these two models have the same architecture, the student model can often outperform the teacher model.
3. We use 8 GPUs.
